### ISSUE TITLE
Droplet Import doesn't set everything 

### ISSUE THREAD

**Issue by mzupan** [terraform-providers/terraform-provider-digitalocean#76]
### Terraform Version

Terraform v0.11.3

### Affected Resource(s)
Please list the resources as a list, for example:
- digitalocean_droplet

## Command Run

```
$ terraform import digitalocean_droplet.db-ref-nyc3-04 83792719
digitalocean_droplet.db-ref-nyc3-04: Importing from ID "83792719"...
digitalocean_droplet.db-ref-nyc3-04: Import complete!
  Imported digitalocean_droplet (ID: 83792719)
digitalocean_droplet.db-ref-nyc3-04: Refreshing state... (ID: 83792719)

Import successful!

The resources that were imported are shown above. These resources are now in
your Terraform state and will henceforth be managed by Terraform.
```

### Expected Behavior
I should be able to run `terraform apply -target  digitalocean_droplet.db-ref-nyc3-04` and not get any changes

### Actual Behavior

```
-/+ digitalocean_droplet.db-ref-nyc3-04 (new resource required)
      id:                   "83792719" => <computed> (forces new resource)
      image:                "" => "ubuntu-16-04-x64" (forces new resource)
      resize_disk:          "" => "true"
      ssh_keys.#:           "" => "1"
      ssh_keys.0:           "" => "895316"
```

We are missing 

- image
- resize_disk
- ssh_keys

**Comment by andrewsomething** [comment]
Thanks for the report. There seem to be a few things to untangle here:

* `image` - Could someone experiencing this provide any more information? I've been unable to reproduce this myself.

* `resize_disk` - This is a non-API attribute, so we should set this to the default value locally. I've addressed this in #95 

* `ssh_keys` - The path forward here is a bit less clear, and I'd love to hear how others would expect this to behave. The DigitalOcean API does not return the SSH keys initially added to the Droplet at create time as part of the Droplet resource nor can SSH keys be added to or removed from an existing Droplet via the DigitalOcean API. My initial feeling is that this should be documented as a known limitation. For an imported Droplet, we can not know if SSH keys have been added or removed manually. If the SSH keys must be managed via Terraform, a new resource will need to be created. If not, then the `ssh_keys` attribute should be excluded in the `tf` config file. Any other opinions?
